index.js

```diff
@@ -1,17 +1,57 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = []
+    this.length = this.items.length
+  }
 
-  add(item) {}
+  add(item) {
+    if (this.length = 1) {
```

Comment:
hello from berlin :) i was just checking the PR's and i see here you write only one "=" that and assignment and not a check and for checking if this.length is = to 1, you have to write "=="
have a nice time in sunny SP :)